src/timedate/timedated.c

```diff
@@ -735,6 +743,9 @@ static int method_set_time(sd_bus_message *m, void *userdata, sd_bus_error *erro
         assert(m);
         assert(c);
 
+        if (c->slot_job_removed)
+                return sd_bus_error_set(error, BUS_ERROR_AUTOMATIC_TIME_SYNC_ENABLED, "Previous request is not finished, refusing.");
```

Comment:
> “Reject Set time until the NTP is fully disabled”.

@rahulmah Thank you for the suggestion. But, the previous request may be `set-ntp true`, that is, enabling...
The condition hits only when `set-ntp` and `set-time` (or equivalent bus APIs) are called sequentially. Of course, there exists a possibility that the callers are different, but it is really rare, I guess. So, the meaning of 'Previous request' is clear. WDYT?